src/content/docs/style-guide/writing-guidelines/user-related-language-guidelines.mdx

```diff
@@ -90,32 +90,37 @@ We recommend linking to the [explanation of default groups](/docs/accounts/accou
 
 ## General philosophy for user permissions/roles language [#user-language-philosophy]
 
-Our general philosophy is summed up in these points:
+First, it will help you to understand the relationship between groups, roles, and capabilities in the new New Relic One user model (sometimes referred to as v2 users). We recommend reading [this explanation of how these concepts relate](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#type-role-group). 
 
-### Document role-related requirements
+The following explanations about our permissions documentation philosophy applies for documenting our New Relic One user model, not our original user model.
 
-For most docs (with exception of API docs, explained in this section), we should only document [role-related requirements](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#standard-roles). If there is a specific [standard role](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#standard-roles) required to use a feature/functionality, explain that in a requirements section so that customers will understand what user role they need to use that feature.
+### Don't document user type or group restrictions 
 
-This approach means that our user-related restriction mentions should be fairly infrequent, as there are only a few features that are gated by a specific standard role. These apply mainly to the organization and user management areas, such as the ability to manage users or manage billing).
+We don't document restrictions related to user type (basic vs. full user) or default groups. Some information about why that is: 
 
-**API docs exception:** Because APIs are not as obvious and transparent as the product UI, we should document any requirements that SMEs think are relevant, which may also include user type. Attempt to list any API requirements in a single location, and not spread them throughout the docs.
+* **Basic vs. full users:** We don't document restrictions related to user type (basic vs. full). This is because things that are off limits to basic users in the product are also places they are given upsell notes. And because those limitations are so widespread, we would be documenting that everywhere. **API docs exception:** Because APIs are not as obvious and transparent as the product UI, we should document any requirements that SMEs think are relevant, which may include user type. Attempt to put any requirements for using an API in a single location, and not spread them throughout the docs.
```

Comment:
I'm not actually sure that we need such detailed explanations in this doc. 

In general, when I'm looking at a style guide, I want to know what to do, I don't particularly care about the why.

Reply:
i think it helps to have the philosophy, for anyone who wants to know the "why." Think that helps cement things better than just a list of "do this" type stuff.